OpenEdXMobile/res/layout/fragment_find_courses.xml

```diff
@@ -16,6 +16,18 @@
         android:listSelector="@android:color/transparent"
         tools:listitem="@layout/row_course_list" />
 
+    <TextView
+        android:id="@+id/no_courses"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textAlignment="center"
+        android:text="@string/no_available_courses"
+        android:textSize="24dp"
```

Comment:
Text sizes should always be in `sp`. Also, instead of hardcoding the textSize here, you can use `@dimen/edx_xx_large`.